packages/cms/src/components/editors/SimpleVisualizationEditor.jsx

```diff
@@ -102,7 +105,12 @@ class SimpleVisualizationEditor extends Component {
           return `\n  "${k}": \`${fixedUrl}\``;
         }
         else {
-          return `\n  "${k}": "${object[k]}"`;
+          if (k === "columns") {
+            return `\n "${k}": [${object[k].map(d => `"${d}"`).join()}]`;
```

Comment:
Could you please change my `join()` hackjob into the `JSON.stringify()` that was suggested today? 
```suggestion
            return `\n "${k}": ${JSON.stringify(object[k])}`;
```